workers/tasks/export/html.tsx

```diff
@@ -206,17 +221,20 @@ const renderFrontMatter = ({
 						{attributions.map((attr, index) => {
 							const {
 								user: { fullName },
-								affiliation,
 							} = attr;
-							const affiliationNumber =
-								affiliation && affiliations.includes(affiliation)
-									? 1 + affiliations.indexOf(affiliation)
-									: null;
+							const affs = getAffiliations(attr);
 							return (
 								<span className="name" key={index}>
 									{fullName}
-									{affiliationNumber && <sup>{affiliationNumber}</sup>}
-									{index < attributions.length - 1 && ', '}
+									{affs?.length > 0 &&
```

Comment:
You could skip this check since if `affs.length === 0` the `map()` will just not render anything.